src-pwa/pwa-flag.d.ts

```diff
@@ -0,0 +1,9 @@
+// THIS FEATURE-FLAG FILE IS AUTOGENERATED,
+//  REMOVAL OR CHANGES WILL CAUSE RELATED TYPES TO STOP WORKING
+import "quasar/dist/types/feature-flag";
+
+declare module "quasar/dist/types/feature-flag" {
+  interface QuasarFeatureFlags {
+    pwa: true;
+  }
+}
```

Comment:
We should not commit this file and `store-flag.d.ts`

Include these in `.gitignore`

/src-pwa/pwa-flag.d.ts
/src/store/store-flag.d.ts

Reply:
@paulyves Remove these files first then add them in a commit.